scientist-softserv/webstore
# 264 create the po

# Story
we need to be able to create purchase orders from the webstore.

# Expected Behavior Before Changes
- we could not create a purchase order from the webstore

# Expected Behavior After Changes
- a purchase order is created for the respective sow when it's accepted

# Screenshots / Video

https://user-images.githubusercontent.com/29032869/228003043-75c291f6-9400-4d36-a6ad-62d7895ca0ac.mp4

# testing instructions
- create, or find, a webstore request with an sow
- accept the sow from the webstore
- make sure that when the page refreshes, the expected PO document is there
